pkgs/by-name/ap/aprx/package.nix

```diff
@@ -0,0 +1,49 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, installShellFiles
+, perl
+}:
+
+stdenv.mkDerivation {
+  pname = "aprx";
+  version = "2.9.1-unstable-2021-09-21";
+
+  src = fetchFromGitHub {
+    owner = "PhirePhly";
+    repo = "aprx";
+    rev = "2c84448fe6d897980234961a87ee4c1d4fad69ec";
+    sha256 = "sha256-01PB7FaG8GmPm1U15/3g1CfQwdYmf3ThZFdVh2zUAl4=";
+  };
+
+  nativeBuildInputs = [ installShellFiles ];
+
+  buildInputs = [ perl ];
+
+  CFLAGS = "-fcommon -O2";
+
+  configureFlags = [ "--with-erlangstorage" ];
+
+  installPhase = ''
```

Comment:
What is wrong with default `make install`? https://github.com/PhirePhly/aprx/blob/2c84448fe6d897980234961a87ee4c1d4fad69ec/Makefile.in#L118C1-L118C8

Reply:
Something like this should work:
```nix
  configureFlags = [
    (lib.withFeature true "erlangstorage")
    "--sbindir=$(out)/bin"
    "--sysconfdir=$(out)/etc"
    "--mandir=$(out)/share/man"
  ];

  makeFlags = [ "INSTALL=install" ];

  preInstall = ''
    mkdir -p $out/bin $out/share/man/man8 $out/etc
  '';
```

Reply:
This works and was incorporated into the most recent fixup